docs/conf/portland/2022/cfp-email-templates.rst

```diff
@@ -0,0 +1,298 @@
+:template: {{year}}/generic.html
+:orphan:
+
+Speaker email templates
+=======================
+
+.. Make this whole file conditional
+
+{% if cfp_variables['print_templates'] and not flagpostconf %}
+
+.. contents::
+   :local:
+   :depth: 1
+   :backlinks: none
+
+.. note:: These templates use the following types of variables:
+
+      * Standard WTD conference variables {% raw %}``{{tz}}``{% endraw %} (which are expanded)
+      * Hidden environment variables {% raw %}``{{cfp_variables['upload']}}``{% endraw %} (which are expanded)
+      * Pretalx variables which are only expanded in Pretalx {% raw %}``{speakers}``{% endraw %}
+
+
+CFP - Acceptance template
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tickets:
+   {{cfp_variables['ticket']}}
+
+--------------------------------------
+
+Subject:
+   WTD {{city}} {{year}} -- talk accepted!
+
+::
+
+   Hi {speakers},
+
+   The Write the Docs {{city}} talk selection committee has finished the review process and we'd love it if you could join us as a speaker!
+
+   Write the Docs {{city}} is held **online {{date.short}}** in {{tz}}.
+
+   We think your talk: '{proposal_title}' would be a great fit for the conference. We'd love to have you prepare it for a **30-minute** time slot.
+
+   Please click this link to confirm your attendance:
+
+   ☐ {confirmation_link}
+
+   Please **reply to this email as soon as possible** if you need to let us know either of the following
+
+   ☐ If you have a strong preference for presenting on Monday or Tuesday, or in the morning or afternoon due to timezones or other restrictions.
+   ☐ If you already purchased a ticket so we can issue you a refund.
```

Comment:
Just to check, is this an intentional non-filled square? Because when I saw it, it looked like a misrendered emoji.

Reply:
Yeah, I was going for checkbox, but I could go back to something like `- [ ] ` or anything else